tests/common/snappi/common_helpers.py

```diff
@@ -100,15 +91,68 @@ def get_egress_lossless_buffer_size(host_ans):
     egress_lossless_pool = buffer_pools[profile_name]
     return int(egress_lossless_pool['size'])
 
+def get_lossless_buffer_size(host_ans):
+    """
+    Get egress lossless buffer size of a switch, unless an 8102 switch,
+    in which case, get the ingress lossless buffer size
+    Args:
+        host_ans: Ansible host instance of the device
+    Returns:
+        total switch buffer size in byte (int)
+    """
+    config_facts = host_ans.config_facts(host=host_ans.hostname,
+                                         source="running")['ansible_facts']
+    is_8102 = True if '8102' in host_ans.facts['platform'] else False
+
+    if "BUFFER_POOL" not in config_facts.keys():
+        return None
+
+    buffer_pools = config_facts['BUFFER_POOL']
+    profile_name = 'ingress_lossless_pool' if is_8102 else 'egress_lossless_pool'
+
+    if profile_name not in buffer_pools.keys():
+        return None
+
+    lossless_pool = buffer_pools[profile_name]
+    return int(lossless_pool['size'])
+
+
+def get_port_prio_ingress_dropped_packets(duthost, phys_intf, prio):
+    """
+    Get number of ingress packets dropped on a specific priority 
+    of a physical interface
+    Args:
+        host_ans: Ansible host instance of the device
+        phys_intf (str): Name of physical interface ex. Ethernet4
+        prio (int): Priority group to check ex. 4 
+    Returns:
+        total number of dropped packets (int)
+    """
+    oid_cmd = "sonic-db-cli " \
+    "COUNTERS_DB HGET COUNTERS_QUEUE_NAME_MAP " + phys_intf + ":" + str(prio)
+    oid_out = duthost.command(oid_cmd)
+    oid_str = str(oid_out["stdout_lines"][0] or 1)
+
+    if oid_str == "1":
+        return None
+
+    cmd = "sonic-db-cli COUNTERS_DB HGET COUNTERS:" + oid_str + \
+    " SAI_QUEUE_STAT_DROPPED_PACKETS"
```

Comment:
Is SAI_QUEUE_STAT_DROPPED_PACKETS used to track ingress packet drops? @neethajohn